plcc/src/plcc/Main.java

```diff
@@ -2401,6 +2417,15 @@ else if (Settings.get("plcc_S_search_similar_method").equals(Similarity.SIMILARI
             System.exit(0);
         }
         
+        if(Settings.getInteger("plcc_I_debug_level") > 0) {
+            for (MolContactInfo mol : cInfo){
+                String mci = mol.toString();
+                String MolA = mol.getMolA().toString();
+                String MolB = mol.getMolB().toString();
+                System.out.println("A ist " + MolA + ". B ist " + MolB);
```

Comment:
In the long term we should create a function for the debug prints. But for now please start debug messages with '[DEBUG LV X]' where X is the intended debug level, in this case 1.

Reply:
I can recommend log4j.